CHANGELOG.md

```diff
@@ -58,7 +58,7 @@ Changes by Version
         If you don't run the migration script, nothing will break, the system will used the old table 
         `operation_names` and set empty `spanKind` in the response.  
         Steps to get the updated functionality:
-        1.  You will need to run below command on the host you can use `cqlsh` to connect the the cassandra contact
+        1.  You will need to run below command on the host you can use `cqlsh` to connect the cassandra contact
          point
```

Comment:
remove this line

Reply:
So should I remove this line or change it? I don't think removing it is a right approach, it maybe cause some confusion.

Reply:
if you apply my previous suggestion, this standalone word "point" should be removed